pyo3-introspection/src/stubs.rs

```diff
@@ -48,5 +48,36 @@ fn class_stubs(class: &Class) -> String {
 }
 
 fn function_stubs(function: &Function) -> String {
```

Comment:
I think it might be nice to check that vararg and kwarg have no default as part of this function?

Reply:
I am a bit reluctant, this function is serializing the input data and imho it's fine to assume that is it valid. If not, we might need to do more checks like validate module/class/function names... But if you feel this case deserve a check I am happy to add it.

Reply:
I have added a new `VariableLengthArgument` type without the `default` field, this prevents the need for this check.